danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/BitBucketServer.kt

```diff
@@ -175,25 +175,25 @@ data class BitBucketServerCommentDetail(
  * @property state The state of the task (e.g. "OPEN")
  * @property author The author of the comment
  */
-@JsonClass(generateAdapter = true)
+@Serializable
 data class BitBucketServerCommentTask(
-        val id: Int,
-        @Json(name = "createdDate")
-        val createdAt: Long,
-        val text: String,
-        val state: String,
-        val author: BitBucketServerUser
+    val id: Int,
+    @SerialName("createdDate")
+    val createdAt: Long,
+    val text: String,
+    val state: String,
+    val author: BitBucketServerUser
 )
 
 /**
  * Properties associated with a comment
  * @property repositoryId The ID of the repo
  * @property issues Slugs of linkd Jira issues
  */
-@JsonClass(generateAdapter = true)
+@Serializable
 data class BitBucketServerCommentInnerProperties(
-        val repositoryId: Int,
-        val issues: Array<String>?
+    val repositoryId: Int,
+    val issues: Array<String>? = null
```

Comment:
Given we can give a default value, can we remove the optionality and put an empty array in case is not present?
So the user has less things to worry about, this can be an empty array or an array of issues, currently can also be null. 
We would remove a state with no changes for the user